routing/pathfind.go

```diff
@@ -167,6 +170,16 @@ func newRoute(sourceVertex route.Vertex,
 					lnwire.TLVOnionPayloadOptional,
 				)
 			}
+
+			// If the receiver supports TLV onion payloads and we
+			// have a payment address from the invoice, we'll go
```

Comment:
If the payment address signals mpp, what signals moon amp then in the invoice?

Reply:
related to my other comment re: feature bits, but we will likely have another feature that signal moon amp